### PR TITLE
Add temperature state (WM-13)

### PR DESCRIPTION
**Pull request by akuntaniguchi**
- Add temperature state variable
- Use OpenWeather API to retrieve temperature
- Fix environment variable section in README

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -4,20 +4,20 @@ import 'semantic-ui-css/semantic.min.css';
...
+  // State
+  const [temperature, setTemperature] = useState('')
 
-  // Function to set the location state value
-  const locationAddHandler = (location: string) => {
-    setLocation(location)
-  };
+  // Function to set the temperature state value
+  const temperatureAddHandler = (temperature: string) => {
+    console.log(temperature)
```
Comment (Kojack8) [inline src/App.tsx]: Did you mean to leave this .log in? Everything else looks good

Comment (akuntaniguchi) [reply src/App.tsx]: Removed! Had that statement to debug

**Comment by Kojack8** [comment]
+1

**Review by Kojack8 — APPROVED** [review]
+1

**Pull request merged**